src/libcollections/btree/map.rs

```diff
@@ -276,6 +344,14 @@ pub struct OccupiedEntry<'a, K: 'a, V: 'a> {
 
 impl<K: Ord, V> BTreeMap<K, V> {
     /// Makes a new empty BTreeMap with a reasonable choice for B.
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
I've been putting a

    /// Basic usage:

line before the first example. Since you're going to be doing a few of these, mind putting this in? If it's too much trouble, I can also do it later.